pkg/machine/ignition.go

```diff
@@ -423,61 +424,67 @@ func getCerts(certsDir string, isDir bool) []File {
 		files []File
 	)
 
-	certs, err := ioutil.ReadDir(certsDir)
 	if isDir {
-		if err == nil {
-			for _, cert := range certs {
-				b, err := ioutil.ReadFile(filepath.Join(certsDir, cert.Name()))
+		err := filepath.Walk(certsDir, func(path string, info os.FileInfo, err error) error {
```

Comment:
Run the code through gofmt -w -s

Reply:
Huh, weird. This command doesn't do anything. If I'll remove the new line, it gets automatically added back on saving in VS Code or by running the autoformatter.  
I also get the error when running `make lint` but have no idea how to fix it as it seems to be against `gofmt`.